wiki/UI-component/Video.md

```diff
@@ -39,6 +39,147 @@ The [[Video]] component utilizes the embeddable Kaltura player <sup>[[r2](#resou
 
 <hr width="40%" />
 
+## 1. Data & content
```

Comment:
This I believe is starting to get into the territory of the video service in my opinion. Which we don't have a functional specs for because it kept getting put on the back burner.

This appears to me to be a list of data provided by the Kaltura API, instead of the data and content model of this component. Am I understanding your intent here correctly?

Reply:
list of data provided by the Kaltura API and content model of this component seems related. Maybe I can restructure to focus / lead with on content of this component.